### DISCUSSION TITLE
i cant find cement

### DISCUSSION THREAD

**Discussion by thisisforaskingquestions** [nroutasuo/level13#315] [Q&A] [answered]
i've gone down to level 4, explored every level from 13-4 entirely, but i just cant find a way to get cement or concrete, i know there are cement mills but i dont have a clue how to make them

**Comment by nroutasuo** [comment, marked as answer]
Sorry for the very late answer, I hope you found them eventually, but just in case anyone else is looking: Concrete is unlocked by an upgrade which requires blueprints that are found around the 6th camp.
